modules/admin_manual/pages/enterprise/file_management/files_lifecycle.adoc

```diff
@@ -0,0 +1,126 @@
+= File Lifecycle Management
+:toc: right
+
+Manage the lifecycle of files within ownCloud to keep your working structure clean, organized and up to date. Choose an archiving and expiration policy that suits your workflow and select an appropriate archive destination to store your old files. Policies can be configured to include and exclude certain files, users
+and groups from the archiving actions. 
+
+== Installation & Configuration
+
+=== Upload Time Tracking
```

Comment:
This is confusing.
First you talk about `File Lifecycle Management` then about `Upload Time Tracking`. Please stay consistant.
Second, starting with 10.5 you have all Enterprise Apps included in the tar file, so nothing needs to be uploaded. Therefore consider renaming the section title `== Installation & Configuration`.

Reply:
?? Lifecycle Management is about Upload Time Tracking

Reply:
If needed have a „Installation“ section...
Then, Upload Time Tracking is not an app. Therefore nothing to install.
You may start with `== Prerequisites` and describe what is necessary to run this app successfully, like enable it (or as I highlighted below that Impersonate may be necessary to be enabled too).
Then you add a section like `==Configuration` to describe config topics ect.
Remember that readers have to follow your thoughts and a clear path.

Reply:
I am not sure if Lifecycle is part of the Complete Tarball. I would expect a market:install.

Please feel free to re structure this PR.

This is the Output from DEV to give a good start for documentation.